src/live_restore/live_restore_fs.c

```diff
@@ -246,18 +248,94 @@ __live_restore_fs_find_layer(WT_FILE_SYSTEM *fs, WT_SESSION_IMPL *session, const
     return (0);
 }
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-parameter"
```

Comment:
Is this still necessary? I don't see an unused parameter in here. If we still need it consider `WT_UNUSED` instead of the pragma